inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java

```diff
@@ -379,17 +385,18 @@ public Boolean update(SinkRequest request, String operator) {
         }
 
         SinkStatus nextStatus = null;
-        boolean streamSuccess = StreamStatus.CONFIG_SUCCESSFUL.getCode().equals(streamEntity.getStatus());
-        if (streamSuccess || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())) {
+        boolean enableConfig = StreamStatus.CONFIG_SUCCESSFUL.getCode().equals(streamEntity.getStatus())
+                || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus());
+        if (enableConfig || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())) {
```

Comment:
`enableConfig || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())` change to `enableConfig`